generator.yaml

```diff
@@ -22,6 +22,14 @@ operations:
   StopHyperParameterTuningJob:
     operation_type: Delete
     resource_name: HyperParameterTuningJob
+  DescribeModelQualityJobDefinition:
```

Comment:
these changes seem unrelated ?

Reply:
These were required changes from v.0.4.0 since those changes were never merged in these are still required changes in v.0.6.0